resident/resident-service/src/main/java/io/mosip/resident/service/impl/ResidentServiceImpl.java

```diff
@@ -787,10 +787,12 @@ public Tuple2<Object, String> reqUinUpdate(ResidentUpdateRequestDto dto, JSONObj
 		String eventId = null;
 		ResidentTransactionEntity residentTransactionEntity = null;
 		try {
+			String sessionUin = null;
```

Comment:
If the variable is not used outside, create the declaration inside the below if condition.

Reply:
validateAuthIndividualIdWithUIN(dto.getIndividualId(), dto.getIndividualIdType(), mappingJsonObject,
					demographicIdentity, sessionUin);
					
it's used here also.